### ISSUE TITLE
App crashes when I open a video 

### ISSUE THREAD

**Issue by miketester10** [dayanch96/YTLite#183]
### Describe the bug

```yaml
When I open the first video in the home list, the application crashes.

Youtube version: 19.36.1
YTPlus version: 5.0.2

iPhone 15
iOS 17.6.1
```

### Steps to reproduce

```yaml
1. Open app
2. Play first video
```

### Post Requisites

- [X] I have checked that a similar issue has not been reported before.
- [X] Verified the relevance of the project and the version you are using.
- [X] Provided a detailed description of the problem and have filled in all required fields.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.
- [X] By submitting this issue, **I confirm that I have filled in all the necessary fields and complied with the guidelines. Failure to comply may result in the issue being closed without discussion, and repeated violations may lead to restrict account from further participation.**

**Comment by dayanch96** [comment]
More info required

**Comment by Chimbul** [comment]
The only thing I found so far that will crash the app is if you enable "Display segments in the feed" in SponsorBlock settings.

![image](https://github.com/user-attachments/assets/f6ab9d07-d3c6-43b0-88ae-e0693f857d94)

**Comment by dayanch96** [comment]
So it’s not ‘Open video’. Its preview video issue. I fixed it already for 5.1 beta, update coming soon

**Comment by dayanch96** [comment]
Fixed here: https://github.com/dayanch96/YTLite/releases/tag/v5.1
